### ISSUE TITLE
[Critical] UIKit 3.5 fails to build with dart sass

### ISSUE THREAD

**Issue by Skayo** [uikit/uikit#4210]
### UIkit Version

3.5.0

### Actual Behavior

I think this is a similar issue to #3832 (*less* syntax used instead of *scss*)

```
Error: expected selector.
    ╷
214 │     .uk-width-xxlarge\@s when ($deprecated = true){ width: $width-2xlarge-width; }
    │                               ^
    ╵
  node_modules\uikit\src\scss\components\width.scss 214:31  @import
  node_modules\uikit\src\scss\components\_import.scss 76:9  @import
  node_modules\uikit\src\scss\uikit-theme.scss 7:9          @import
  assets\scss\site.scss 17:9                                root stylesheet
 ERROR  Command failed with exit code 65.
```

### Expected Behavior

Build without problems

### Steps to Reproduce the Problem

*Basically just follow the guide here: [UIKit Docs](https://getuikit.com/docs/sass)*
1. `$ npm install uikit sass`
2. `$ sass site.scss site.css`

**Comment by Skayo** [comment]
At this point I should note that it would probably be good idea to use a real sass compiler when testing instead of your less-to-sass-converter thingy - this bug would not have been missed if you did that! (I think)

**Comment by janschoenherr** [comment]
Fixed in 3.5.1
